packages/light-client/test/unit/webEsmBundle.browser.test.ts

```diff
@@ -1,7 +1,7 @@
-/* eslint-disable @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-member-access */
 import {expect, describe, it, vi, beforeAll} from "vitest";
 import {sleep} from "@lodestar/utils";
-import {Lightclient, LightclientEvent, utils, transport} from "../../dist/lightclient.min.mjs";
+import {Lightclient, LightclientEvent, utils, transport} from "../../src/index.js";
```

Comment:
I don't think this is correct as we are no longer testing the web bundle. The problem with this is as soon as you run `yarn check-types` locally, this file will fail `yarn lint` (this is why it does not fail in CI run type checks and lint separately).

I opened an issue for this a while ago https://github.com/ChainSafe/lodestar/issues/6726, we need to look into properly fixing this and not fail lint locallly.

cc @nazarhussain

Reply:
well i just carried the rebase change, someone who has more insight (you or nazar) can tackle it